guide/02-api-overview/release_notes_210.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<ul><li><a href="https://github.com/python/samples" target="_blank">Advanced Machine Learning</a></li><li>* 2d Computer Vision</li><li>Object Detection</li><li class="ql-indent-1"><a href="https://github.com/python/samples/detecting-palm-trees-using-deep-learning" target="_blank">Detecting Palm Trees using Deep Learning</a></li><li class="ql-indent-1"><a href="https://github.com/python/samples/detecting-swimming-pools-using-automated-machine-learning" target="_blank">Detecting Swimming Pools using Automated Deep Learning</a></li><li class="ql-indent-1"><a href="https://github.com/python/samples/traffic-light-detection-on-oriented-imagery" target="_blank">Traffic Light Detection in Oriented Imagery Using ArcGIS Pretrained Model</a></li><li>Feature Extraction</li><li class="ql-indent-1"><a href="https://github.com/python/samples/extracting0-features-and-land-use-land-cover-using-panoptic-segmentation" target="_blank">Extracting features and Land Use Land Cover using Panoptic Segmentation</a></li><li class="ql-indent-1"><a href="https://github.com/python/samples/coastline-extraction-usa-landsat8-multispectral-imagery" target="_blank">Coastline extraction using Landsat-8 multispectral imagery and band ratio technique</a></li><li class="ql-indent-1"><a href="https://github.com/python/samples/flood-inundation-mapping-using-sar-data-and-deep-learning" target="_blank">Flood inundation mapping and monitoring using SAR data and deep learning</a></li><li>Generative Models</li><li class="ql-indent-1"><a href="https://github.com/python/samples/generating-lst-from-multispectral-imagery-using-pix2pix" target="_blank">Generating Land Surface Temperature from multispectral imagery using Pix2Pix</a></li><li class="ql-indent-1"><a href="https://github.com/python/samples/cloud-detector-part1-cloudless-sentinel-&amp;-unsupervised" target="_blank">Generating cloud masks from satellite imagery - Part I</a></li><li><s>Other Imagery Use Cases</s></li><li class="ql-indent-1"><a href="https://github.com/python/samples/cloud-removal-using-pre-trained-deep-learning-model-and-raster-function" target="_blank"><s>Cloud removal using pre-trained deep learning model and raster function</s></a></li><li>Tabular Data Forecasting</li><li class="ql-indent-1"><a href="https://github.com/python/samples/house-price-prediction-using-automl" target="_blank">Property Valuation in King County USA using AutoML and Geoenriched data</a></li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Changed structure!